packages/cli/package.json

```diff
@@ -14,7 +14,7 @@
     "preinstall": "node ./scripts/preinstall.js",
     "test": "jest --env node --verbose --bail",
     "test-unit": "pnpm test test/unit/",
-    "test-cli": "rimraf test/fixtures/integration && pnpm test test/integration.test.ts",
+    "test-cli": "rimraf test/fixtures/integration && pnpm test test/integration-1.test.ts test/integration-2.test.ts test/integration-3.test.ts",
```

Comment:
I think `test-cli` could be renamed to `test-e2e` since it is creating deployments in many cases and then you wouldn't need to touch `chunk-tests.js`